common/source/util/diagnostic/main.cpp

```diff
@@ -218,9 +235,9 @@ int main(int argc, char *argv[]) {
   // preliminary information about all or just the one specified
   if ((!probe && device_name == NULL) || (probe && device_name != NULL)) {
     if (scan_devices(device_name) == 0) {
-      printf("\nDIAGNOSTIC_PASSED\n");
+      printf("\nBSP DIAGNOSTIC_PASSED\n");
```

Comment:
BSP or ASP?

Reply:
aocl is using BSP , so for consistency using BSP for now in message